src/functions/content-types/product/item.ts

```diff
@@ -94,6 +88,6 @@ function calculateValueFields(data) {
   // currently USD is the only unit available.
   const { packagePrice, countPerPackage } = data;
   let pricePerItemUSD = packagePrice / countPerPackage;
-  pricePerItemUSD = parseFloat(pricePerItemUSD.toFixed(2));
+  pricePerItemUSD = Math.round(pricePerItemUSD * 100) / 100;
```

Comment:
**question:** @madcalf I think I was wrong in the comments on the other PR.  `.toFixed(n)` will round so either way should be fine.  Go with the one that reads cleaner to you?

Reply:
@madcalf sounds good!